metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -1697,7 +1705,7 @@ public RollbackResult deleteAspect(String urn, String aspectName, @Nonnull Map<S
         latest.setCreatedOn(survivingAspect.getCreatedOn());
         latest.setCreatedBy(survivingAspect.getCreatedBy());
         latest.setCreatedFor(survivingAspect.getCreatedFor());
-        _aspectDao.saveAspect(latest, false);
+        _aspectDao.saveAspect(latest, false, null);
```

Comment:
Confused on why some of these are passing null through even though the method signature was updated to have the extraIngestParams